frontend/src/components/admin/TicketManagement.jsx

```
@@ -1,380 +1,214 @@
-      // setTickets(response.data);
+      const response = await ticketAPI.getTickets();
+      
+      // 응답 데이터 구조 분석 및 파싱
+      let ticketData = [];
+      if (response.data) {
+        if (Array.isArray(response.data)) {
+          ticketData = response.data;
+        } else if (response.data.data && Array.isArray(response.data.data)) {
+          ticketData = response.data.data;
+        } else if (response.data.data && response.data.data.content && Array.isArray(response.data.data.content)) {
+          ticketData = response.data.data.content;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)